src/SumoCoders/FrameworkCoreBundle/Resources/assets/coffee/framework.coffee

```diff
@@ -150,26 +189,17 @@ class Framework extends DefaultObject
 
     $(this).tab('show')
 
-  _calculateActionsWidths: ->
-    $('.actions li a').each(->
-      $this = $(@)
-      $this.attr('data-width', $this.width())
-      $this.width(0)
-      $this.hover(->
-        $this.width($this.data('width') + 20)
-      ,->
-        $this.width(-20)
-      )
-    )
+  toggleSearchBar: ->
+    $('.search-box').toggleClass('open')
```

Comment:
haakjes rond 'open' zij niet nodig.

over het algemeen is er nogal wat code die gewoon JS syntact gebruikt ipv coffee syntax.